src/lib/journey/stages/_utilities/stage.utilities.ts

```diff
@@ -0,0 +1,38 @@
+import { get } from 'svelte/store';
+
+import { configuredJourneys } from '$journey/config.store';
+
+import type { StageJourneyObject } from '$journey/journey.interfaces';
+
+export function captureLinks(linkWrapper: HTMLElement, currentJourney: StageJourneyObject) {
```

Comment:
This was an important function to provide for OOTB behavior. The default journeys from ID Cloud are not named with a stage and therefor go through the generic stage component. The links that come from the Page Node's "Page Description" to forgot password and such are exactly that, links. So, I need to capture them, and internally handle the rendering of the selected journey, rather than navigate to the link's default `href` value (e.g. `#/service/ResetPassword`).

Just to reiterate, from the Widget's perspective, we can't rely on navigation since it's installed in a foreign app and making any assumptions about the parent app's route handling would be dangerous.

Reply:
I agree with the way you've done it here. We'll need to make sure the documentation on this behaviour is quite explicit though, as I think this could get quite confusing to a developer otherwise!